ansible/pbTestScripts/testScript.sh

```diff
@@ -78,6 +78,10 @@ checkVars()
                 echo "Can't find vagrant-disksize plugin, installing . . ."
                 vagrant plugin install vagrant-disksize
         fi
+	if [[ ! -f ~/.ssh/id_rsa ]]; then
```

Comment:
We probably shouldn't be generating a new key for the user who runs this. I suggest we create a new "one-time" key for each run of the script and store it in a suitable location (probably the shared directory) and use that instead.